README.md

```diff
@@ -69,10 +69,17 @@ For user space, we need user application debuginfo like `redis-debuginfo-2.8.19-
 For *redhat*\* linux version, we can install as the following:
 
 ```bash
-yum install yum-utils #for debuginfo-install
-yum install systemtap
-yum install kernelname-devel-version
-debuginfo-install kernelname-version
+yum install -y redhat-lsb
+
+yum install -y kernel-devel-`uname -r`
+
+yum install -y http://debuginfo.centos.org/`lsb_release -sr|cut -d. -f1`/`uname -m`/kernel-debuginfo-common-`uname -m`-`uname -r`.rpm
```

Comment:
We can tell users to get debuginfo  package on [http://debuginfo.centos.org](http://debuginfo.centos.org) if they are in Centos